packages/TelegramClientTests-Misc.package/TCTMMocks.class/class/mockPhotoMessageWith..st

```diff
@@ -0,0 +1,14 @@
+message
+mockPhotoMessageWith: aCore
+
+	^ TCCPhotoMessage new
+		chat: self mockPrivateChat;
+		core: aCore;
+		senderID: self mockUser1 id;
+		isOutgoing: false;
+		date: self mockMessageDate;
+		id: self mockMessageID;
+		caption: self mockText;
+		width: 521;
+		height: 265;
```

Comment:
maybe we could use something like self mockPhotoWidth/Height

Reply:
implemented your suggestion